docs/reference_guide.md

```diff
@@ -390,6 +391,15 @@ Default: 4096
 
 This is the maximum number of keys that can be stored in a map. Increasing the value will consume more memory and increase startup times. There are some cases where you will want to: for example, sampling stack traces, recording timestamps for each page, etc.
 
+## 9. Clang Environment Variables
+
+bpftrace compiles programs with clang. Thus environment variable affecting 
```

Comment:
Could you clarify this to say something like "bpftrace parses C header files using libclang". We don't actually use it to compile anything

Reply:
sure